config/config.go

```diff
@@ -512,6 +513,14 @@ type EpochStartDebugConfig struct {
 	ProcessDataTrieOnCommitEpoch bool
 }
 
+// ProcessDebugConfig will hold the process debug configuration
+type ProcessDebugConfig struct {
+	Enabled              bool
+	LogLevelChanger      string
+	GoRoutinesDump       bool
```

Comment:
variables with the same `type` should stay together (optime memory usage) 
`GoRoutinesDump` can be moved near `Enabled` field

Reply:
done